ffi/derive/src/bindgen.rs

```diff
@@ -37,7 +37,7 @@ pub fn gen_ffi_fn(fn_descriptor: &FnDescriptor) -> proc_macro2::TokenStream {
     quote! {
         #[doc = #ffi_fn_doc]
         #[no_mangle]
-        pub unsafe extern "C" fn #ffi_fn_name(#(#self_arg,)* #(#fn_args,)* #output_arg) -> iroha_ffi::FfiResult {
```

Comment:
please keep returning `FfiResult` fieldless enum. How will the caller of the function know if something went wrong and whether they need to poll for error?

Reply:
We don't want the user to poll for error on every FFI call